src/foundry/client/game.d.mts

```diff
@@ -645,8 +642,11 @@ declare global {
         PackageCompendiumData & {
           /** @deprecated since v11 */
           private?: boolean;
-          system?: string;
+          system?: string | undefined;
           type: foundry.CONST.COMPENDIUM_DOCUMENT_TYPES;
+          packageName: BasePackage["_source"]["id"];
+          packageType: BasePackage["type"];
+          id: string;
```

Comment:
This is an update to `game.data`; these properties should not be here, but in the `readonly packs` above which is typed as `CompendiumPacks`. I would strongly suggest starting a `tests/foundry/client/data/collections/compendium-packs.test-d.ts` file so you can see why the updates need to be made to the `CompendiumPacks` class.